docs/pages/admin-guides/management/guides/aws-iam-identity-center.mdx

```diff
@@ -182,11 +173,60 @@ Clicking the "Log In" button for this app takes you to your Identity Center
 SSO start page which you can use to pick a role and connect to your AWS account
 as usual.
 
+## Background: Account Assignments
```

Comment:
I think it would make sense to incorporate this into "How it works", since it's architectural background that a reader can keep in mind while they continue through the setup steps.